src/calculateRentalCost.js

```diff
@@ -28,7 +28,9 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalCost = days * 40;
+
+  return days >= 7 ? totalCost - 50 : days >= 3 ? totalCost - 20 : totalCost;
```

Comment:
```suggestion
  return days >= 7 ? totalCost - 50 : days >= 3 ? totalCost - 20 : totalCost;
```
Don't use ternary operators here. Use if-else if

Also, create variables for 50 and 20. It's not clear what are these numbers